platform_design/lib/widgets.dart

```diff
@@ -94,7 +92,7 @@ class _PressableCardState extends State<PressableCard>
         behavior: HitTestBehavior.opaque,
         onTap: () {
           if (widget.onPressed != null) {
-            widget.onPressed();
+            widget.onPressed!();
```

Comment:
It's possible to replace this with `widget.onPressed?.call()` and take out the if-statement, but this way is better for readability.

Reply:
I think we should change it. People will get to learn the use of the `?` operator more, like I did.